workflow/applications/applications.py

```diff
@@ -156,7 +152,7 @@ def source_configs(self, run: str = "gfs", log: bool = True) -> Dict[str, Any]:
         configs = dict()
 
         # Return config.base as well
-        configs['base'] = self.conf.parse_config('config.base')
+        configs['base'] = conf.parse_config('config.base')
```

Comment:
This should also be based on run
```suggestion
        configs['base'] = conf.parse_config('config.base', RUN=run)
```

Reply:
Fixed.